lib/publiccloud/instance.pm

```diff
@@ -429,7 +429,10 @@ sub softreboot {
     }
 
     $self->ssh_assert_script_run(cmd => 'sudo /sbin/shutdown -r +1');
-    sleep 60;    # wait for the +1 in the previous command
+    # wait for the +1 in the previous command
+    sleep 60;
+    # add more time to wait for the system to turn off for SLES4SAP test cases
+    sleep 120 if (get_var('PUBLIC_CLOUD_SLES4SAP'));
```

Comment:
I addition, please, a kind comment:  
this PR I see is only adding this more 120sec for SAP cases, but plese consider we that the _timeout_ in `migration` module could be tuned/increased by changing value of `PUBLIC_CLOUD_REBOOT_TIMEOUT` parameter in test settings or clone_job cmd line.

Reply:
The `sleep 120` is for `wait for the system to turn off`.
Set PUBLIC_CLOUD_REBOOT_TIMEOUT=1200 for this test case.